app/components/menubar/newMenuBar.js

```diff
@@ -0,0 +1,350 @@
+                }
+                imageTitle={"Rocket.Chat"}
+                brandName={"Rocket.Chat Community"}
+                height={30}
+                width={132}
+              />
+            </Navbar.Brand>
+          </Offcanvas.Header>
+          <Offcanvas.Body>
+            {nav_Items?.map((nav_Item) =>
+              nav_Item.url ? (
+                <div key={nav_Item.id}>
+                  <Row
+                    className={`${styles.dropdown} d-flex flex-row justify-content-between align-items-center mt-3 `}
+                    onClick={() => {
+                      setDropDown({ show: false, _id: 0 });
+                    }}
+                  >
+                    <Col>
+                      <a
+                        href={nav_Item.url}
+                        className='text-decoration-none fs-4 fw-light text-dark'
+                      >
+                        {nav_Item.label}
+                      </a>
+                    </Col>
+                  </Row>
+                </div>
+              ) : (
+                <div key={nav_Item.id}>
+                  <Row
+                    className={`${styles.dropdown} d-flex flex-row justify-content-between align-items-center mt-3 `}
+                    onClick={() => {
+                      if (dropDown._id === nav_Item.id) {
+                        setDropDown({ show: false, _id: 0 });
+                      } else {
+                        setDropDown({ show: true, _id: nav_Item.id });
+                      }
+                    }}
+                  >
+                    <Col
+                      className={
+                        dropDown._id === nav_Item.id && dropDown.show
+                          ? `${styles.color} fs-4 fw-light`
+                          : 'fs-4 fw-light'
+                      }
+                    >
+                      {nav_Item.label}
+                    </Col>
+                    <Col>
+                      {nav_Item.sub_menus?.data?.length > 1 && (
+                        <span
+                          className={
+                            dropDown.show
+                              ? `${styles.arrowRotate} bg-transparent me-2`
+                              : `${styles.arrow} bg-transparent me-2 `
+                          }
+                        >
+                          <ArrowIcon />
+                        </span>
+                      )}
+                    </Col>
+                  </Row>
+                  {dropDown._id === nav_Item.id && dropDown.show ? (
+                    <div>
+                      <div>
+                        {nav_Item.sub_menus.data.map(
+                          (item, key) =>
+                          (
+                            <>
+                              <div key={key} className={'p-2 fw-medium' + ` ${item.attributes.style === 'disable' ? styles.disabled : ''}`}>
```

Comment:
Could we change the style for disabled and link items? 

![image](https://user-images.githubusercontent.com/61188295/213871132-cffcaacb-2056-40a7-aaf6-481f83663574.png)

In the above image, the `disabled` item (`GSoC 2022`), which serves as a heading for categories (`GSoC 2022 Demo Day`), is not easily distinguishable. 

Another note:

Please keep the Nav Links in the left corner:
![image](https://user-images.githubusercontent.com/61188295/213871450-3faad748-dd5e-4766-830d-7872f4af60c1.png)
 (Something like above)